tests/testthat/_snaps/bump_version.md

```diff
@@ -1,37 +1,5 @@
 # bump_version() works
 
-    Code
```

Comment:
Are these changes intended? Two parallel snapshots was the best I could do to test both CRAN and dev rlang.